src/dom/html/WindowRe.re

```diff
@@ -83,6 +83,7 @@ module Impl (T: { type t; }) => {
   external print : unit = "" [@@bs.send.pipe: t_window];
   external prompt : string => string = "" [@@bs.send.pipe: t_window];
   external promptWithDefault : string => string => string = "prompt" [@@bs.send.pipe: t_window];
+  external requestAnimationFrame : (float => unit) => unit = "" [@@bs.val];
```

Comment:
This doesn't really belong in the Dom API. It's only attached to `window` because that's the global scope for historic reasons. It was originally introduced in the animation-timing standard, but is now part of the animation frames section of the HTML5 standard. Maybe add an `Animation` module and dump it there instead.

Reply:
Sounds good, I'll fix it 👍